src/main/jjtree/net/sf/jsqlparser/parser/JSqlParserCC.jjt

```diff
@@ -635,13 +643,109 @@ DescribeStatement Describe(): {
 
 ExplainStatement Explain(): {
     Select select;
+    List<ExplainStatement.Option> options = null;
 } {
-    <K_EXPLAIN> select = Select()
+    <K_EXPLAIN>
+    options=ExplainStatementOptions()
+    select = Select()
     {
-        return new ExplainStatement(select);
+       ExplainStatement es = new ExplainStatement(select);
+       if(options != null && !options.isEmpty()) {
+         for(ExplainStatement.Option o : options) {
+           es.addOption(o);
+         }
+       }
+       return es;
     }
 }
 
+/**
+ * Postgres supports TRUE,ON,1,FALSE,OFF,0 as values
+ */
+String ExplainOptionBoolean():
+{
+  Token tk = null;
+}
+{
+   // intentionally not supporting 0,1 at the moment
+   [( tk=<K_TRUE> | tk=<K_FALSE> | tk=<K_ON> | tk=<K_OFF> )] // optional
+   {
+    return tk != null ? tk.image : null;
+   }
+}
+
+/**
+ * The output format, which can be TEXT, XML, JSON, or YAML
+ */
+String ExplainFormatOption():
+{
+  Token tk = null;
+}
+{
+   // TODO support Text
```

Comment:
ran into issues with conflicting "TEXT", so i've just left it out atm until someone requests it.